chemicaltoolbox/xchem-deep/xchem_deep_score.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="xchem_pose_scoring" name="XChem pose scoring" version="0.1.0">
+    <description>- perform pose scoring using deep learning</description>
+
+    <requirements>
+        <!--requirement type="package" version="3.0.0">openbabel</requirement-->
+        <!--requirement type="package" version="3.7">python</requirement-->
+        <!-- many other requirements are needed -->
+        <container type="docker">informaticsmatters/deep-app-ubuntu-1604:latest</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    ln -s '$ligands' ligands.sdf &&
+    ln -s '$receptor' receptor.pdb &&
+    python '$__tool_directory__/xchem_deep_score.py' -i ligands.sdf -r receptor.pdb $mock &&
+    cp output.sdf $output
+    ]]></command>
+
+
+    <inputs>
+        <param type="data" name="receptor" format="pdb,mol2" label="Receptor" help="Select a receptor (pdb or mol2 format)."/>
+        <param type="data" name="ligands" format="sdf,mol" label="Ligands" help="Ligands (docked poses) in SDF format)"/>
+        <param type="hidden" name="mock" truevalue="--mock" falsevalue="" label="Mock calculations" help="Use random numbers instead of running on GPU"/>
```

Comment:
```suggestion
        <param type="hidden" name="mock" value="" label="Mock calculations" help="Use random numbers instead of running on GPU"/>
```

Reply:
Seems this is not documented at all, but I think this is correct. So the value is always `""` for the user but is overridden by the test.